saleor/product/models/base.py

```diff
@@ -235,6 +241,7 @@ def get_first_category(self):
 
     def is_available(self):
         # type: () -> bool
+        """Checks if product is available today for customers"""
```

Comment:
I think "today" documents the current implementation rather than the intended use.

```
`True` if product is available for purchase, `False` otherwise.
```